driver/node_test.go

```diff
@@ -0,0 +1,16 @@
+package driver
+
+import (
+	"testing"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+)
+
+func TestMakeMountOptions(t *testing.T) {
+	_, err := makeMountOptions(true, &csi.VolumeCapability_MountVolume{
```

Comment:
Could you please add one more test for normal pattern.

Reply:
Do you mean a pattern for specifying read-only false and rw option? I think it is trivial case and if the pattern fails, all e2e will not pass, so there is not worth adding it.

Reply:
ah, the pattern I mentioned is not normal, I doubt the normal pattern is read-only false and no options. If it is your case, I will add it.

Reply:
What I wanted to say is that "rw" is not specified. But, as you pointed out, all e2e will not pass. I withdraw my point.